lib/index.js

```diff
@@ -198,14 +208,29 @@ class PWMetrics {
   }
 
   findMedianRun(results) {
-    const ttiValues = results.map(r => r.timings.find(timing => timing.id === metrics.ids.TTI).timing);
-    const medianTTI = median(ttiValues);
+    const maxTimingValue = Math.max(...results.reduce((acc, r) => {
+      // Result may have invalid data -> failed run
+      if (r.timings)
```

Comment:
You are right this redundant, the error filter in the AllDone.then function should deal with this.